portfolio/src/main/webapp/index.html

```diff
@@ -8,16 +8,30 @@
   </head>
     <header>
       <h1>JENNIFER JIMENEZ</h1>
+      <div class="logos">
+        <a href="https://www.linkedin.com/in/jen-jimenez" target="_blank"> 
+          <img src="/images/linkedin.png" id="logo"/>
+        </a>
+        <a href="https://github.com/jennifer-jimenez" target="_blank"> 
+          <img src="/images/git.png" id="logo"/>
+        </a>
+        <a href="https://www.instagram.com/prettyoddjen/" target="_blank"> 
+          <img src="/images/ig.png" id="logo"/>
+        </a>
+      </div>
     </header>
     <nav>
       <ul class="navbar">
-        <li>Home</li>
-        <li>Projects & Work</li>
-        <li>Gallery</li>
+        <li><a href="index.html" class="nav_link"><u>Home</u></a></li>
+        <li><a href="projects.html" class="nav_link">Projects</a></li>
+        <li><a href="gallery.html" class="nav_link">Gallery</a></li>
       </ul>
     </nav>
-    <body>
-    <div id="content">
+
+  <body>
+  <div id="wrapper">
+    <div id="text">
```

Comment:
nit: As above, "text" is probably not specific enough for an `id`, since there might eventually be other text on the page that falls outside this structure. Maybe something like "bio" here?